src/CategoryDescription.css

```diff
@@ -0,0 +1,63 @@
+:root {
```

Comment:
I am not too sure here but does this need to be prefixed by the .category-container class or added to a main root css file for colors?

Reply:
I pulled the colors from the Figma file so I think everyone can use it. Moving this to App.css.